.mailmap

```diff
@@ -77,6 +77,7 @@ Florent Manens <florent@beezim.fr> Florent Manens <florent@manens.org>
 Florian Coste <fcoste21@gmail.com> nairolf21
 Florian Marsylle <florian.marsylle@hotmail.fr> <floleblanc@hotmail.fr>
 François Lafont <flafdivers@free.fr> <francois.lafont@crdp.ac-versailles.fr>
+François Lafont <flafdivers@free.fr> lafont <francois.lafont@ac-versailles.fr>
```

Comment:
That's right. There are probably others occurences of this unnecessary "name + email" in this file...